renga_deployer/engines.py

```diff
@@ -498,20 +522,47 @@ def get_host_ports(self, execution):
             return {'ports': []}
 
         else:
-            return {
-                'ports': [{
-                    'specified':
-                    port.port,
-                    'host':
-                    current_app.config[
-                        'DEPLOYER_K8S_CONTAINER_IP'] or pod.items[
-                            0].status.host_ip,
-                    'exposed':
-                    port.node_port,
-                    'protocol':
-                    port.protocol,
-                } for port in service.items[0].spec.ports]
-            }
+            if current_app.config.get(
+                    'DEPLOYER_K8S_USE_INGRESS'):   # pragma no cover
```

Comment:
Can we add tests instead ignoring the code?

--
https://medium.com/@Oskarr3/setting-up-ingress-on-minikube-6ae825e98f82

Reply:
I added tests covering all the ingress parts